admin/constraints/jutsu.php

```diff
@@ -42,6 +42,8 @@
     'counter',
     'immolate',
     'recoil',
+    'delayed_residual',
```

Comment:
How does this effect behave differently from standard residual? I looked through the code in the rest of the files but don't see any mechanical changes